spec/javascripts/components/story/expanded_story/expanded_story_labels_spec.js

```diff
@@ -63,34 +70,41 @@ describe('<ExpandedStoryLabels />', () => {
       it('renders nothing', () => {
         const { wrapper } = setup({ disabled: true }, []);
 
-        expect(wrapper.html()).toBeNull();
+        expect(wrapper.innerHTML).toBe('');
       });
     });
   });
 
   describe('when component is enabled', () => {
     it('allows deleting labels', () => {
       const { wrapper, onRemoveLabel } = setup();
+      const deleteButton = wrapper.querySelector('.react-tags__selected-tag');
 
-      const handleDelete = wrapper.find('ReactTags').prop('handleDelete');
-
-      handleDelete(0);
+      fireEvent.click(deleteButton);
 
       expect(onRemoveLabel).toHaveBeenCalled();
     });
 
     it('allows adding labels', () => {
-      const { reactTags } = setup();
+      setup();
+      const addLabel = screen.queryByText(I18n.t('add new label'));
 
-      expect(reactTags.prop('allowNew')).toBe(true);
+      expect(addLabel).toBeInTheDocument();
     });
   });
 
   describe('<ReactTags />', () => {
     it('renders with the right tags prop', () => {
-      const { labels, reactTags } = setup();
-
-      expect(reactTags.prop('tags')).toEqual(labels);
+      const { wrapper, labels, reactTags } = setup();
+      screen.debug();
```

Comment:
Can we remove this debug?

Reply:
Done!

[eb89ada](https://github.com/Codeminer42/cm42-central/pull/926/commits/eb89ada7aa69055db257af2a4e44879032fe7ecc)